docs/arcadia-users-group/20240305-github-templates/lesson.md

```diff
@@ -0,0 +1,71 @@
+# Introduction to GitHub Templates
+
+GitHub templates provide a structured way to create new repositories, ensuring consistency, efficiency, and adherence to best practices across projects.
+GitHub templates are predefined repository setups that can be used to start new projects with a consistent set of files, folders, and configurations.
+This not only saves time but helps make sure that all projects follow Arcadia's coding and documentation standards.
+Templates typically include everything from README files, installation guides, contribution guidelines, and more.
+
+In this lesson, we will focus on Arcadia's [`python-analysis-template`](https://github.com/Arcadia-Science/python-analysis-template).
+Arcadia also has an [`r-analysis-template`](https://github.com/Arcadia-Science/r-analysis-template), a [`snakemake-template`](https://github.com/Arcadia-Science/snakemake-template), and a [`nextflow-template`](https://github.com/Arcadia-Science/nextflow-template).
+
+## Creating a new repository using a template
+
+### Choosing the Right Template
+
+Depending on the analysis type or the technology stack used in your project, select the appropriate template. 
+For instance, use `r-analysis-template` for R-based analyses, `python-analysis-template` for Python projects, `snakemake-template` for workflows managed with Snakemake, and `nextflow-template` for Nextflow pipelines.
+
+<details>
+<summary>Why isn't there a Python package template?</summary>
+
+We could create one. 
+We haven't generated one yet because there are a ton templates out there that could be used and we haven't created many Python packages yet as an organization.
+If you find yourself in need of a Python template, try out <a href=https://github.com/TezRomacH/python-package-template>an existing template</a> and make note of what works well and what doesn't.
+We can then use that to generate a future organization standard template.
+
+</details>
+
+### Repository Initialization
+
+There are two ways to create a new repo with a template.
```

Comment:
I wonder if we should reframe this as an explicit list of steps, which people will walk through together during the AUG.

Reply:
ya that is my plan, I am going to lead people through this. I won't have the lesson up at that point, I'm going to do a show and follow me demonstration. Since this is my plan, do you think it's ok to leave the lesson as is, or should i re-write it to make it more clear that that is my plan?

Reply:
It might be nice to do a walkthrough style approach, as was done for previous AUGs; esp. for folks who will be reviewing the lesson at a later date! Basically the only change would be to reformat the paragraph as a bulleted / numbered list.

Reply:
done, thank you!